volumes/short_term_counting_program/README.md

```diff
@@ -101,174 +101,187 @@ The City of Toronto retains a traffic counting contractor who conducts data coll
+Traffic counts are conducted ad-hoc, usually on request for a specific project need. As such, the data is not necessarily systematically collected. We do not have comprehensive coverage across time or space. However, it is worth noting that there are now proactive counts being done for areas that have been identified as needing updated data.
 
 TMCs are processed automatically, nightly, once made available from the contractor. ATRs are loaded manually by staff once data files are received from the contractor.
 
 ## Where can I access the data?
 
-Internal to the Transportation Data & Analytics team, data flows from legacy Oracle database, nightly to MOVE (`flashcrow` RDS), and is then replicated to the `bigdata` RDS.
+Internal to the Transportation Data & Analytics team, legacy data flows from legacy Oracle database, nightly to MOVE (`flashcrow` RDS), and is then replicated to the `bigdata` RDS. Newer data is loaded directly into MOVE and then replicated to the `bigdata` RDS.
 
 Look in the `traffic` schema for all ad-hoc data tables.
 
 ## Where can I find what data?
 
-| Study Type // Loading Mechanism | FlowLoad (`traffic.*`)                           | MOVE Loader (`traffic.atr_*`)        | Spectrum API Loader (`traffic.tmc_*`) |
-|---------------------------------|--------------------------------------------------|--------------------------------------|---------------------------------------|
-| Turning Movement Count          | All-time TMC data*                               | n/a                                  | September 2023 to present             |
-| Volume ATR                      | All-time Volume ATRs**                           | n/a                                  | n/a                                   |
-| Speed / Volume ATR              | All-time Speed/Vol ATRs*                         | May 2023 to present                  | n/a                                   |
-| Vehicle Classification ATR      | Classification ATR data from 1985 to May 2023*** | No Classification ATR data loaded*** | n/a                                   |
+Speed Volume Classification counts and Turning Movement Counts are being replicated into Bigdata `traffic` schema from MOVE (FLASHCROW database). Every table that is replicated has a link to internal documentation for the corresponding table on FLASHCROW in the table comment, viewable in table properties in PGAdmin.
+
+Public documentation including data dictionaries are accessible on the Open Data pages:
+
+- SVC: [Traffic Volumes - Midblock Vehicle Speed, Volume and Classification Counts](https://open.toronto.ca/dataset/traffic-volumes-midblock-vehicle-speed-volume-and-classification-counts/) 
+- TMC: [Traffic Volumes - Multimodal Intersection Turning Movement Counts](https://open.toronto.ca/dataset/traffic-volumes-at-intersections-for-all-modes/)
 
-*The `traffic.*` tables contain all-time TMC and Speed/Vol ATR data from all loading mechanisms
+Load Sources Summary
```

Comment:
I'm not following in which table I should be looking up these different columns. Should it come after the "How is the data structured?" section?

Also the asterix notes all got deleted.